Manatee.Trello.WebApi/WebApiClient.cs

```diff
@@ -150,7 +150,10 @@ private static HttpContent GetContent(WebApiRestRequest request)
 				return formData;
 			}
 			var body = TrelloConfiguration.Serializer.Serialize(request.Body);
-			return new StringContent(body);
+			var jsonContent = new StringContent(body);
+            jsonContent.Headers.ContentType = new System.Net.Http.Headers.MediaTypeHeaderValue("application/json");

```

Comment:
Please:

1. Fix formatting (tabs).
2. Add a `using` instead of explicitly listing the namespace.